.github/workflows/test-build-deploy.yml

```diff
@@ -0,0 +1,117 @@
+name: ci
+on:
+  push:
+    branches: [master]
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+** # Tag filters not as strict due to different regex system on Github Actions
```

Comment:
As of Oct 2020, GitHub [does not support](https://github.community/t/using-regex-for-filtering/16427/2)regex for tag filtering. The old regex  `/^v[0-9]+(\.[0-9]+){2}(-.+|[^-.]*)$/` was approximated using the available GitHub [filter patterns](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)

Reply:
Looks good enough to me 👍